src/pathfora.js

```diff
@@ -611,8 +611,9 @@
       if (!total) {
         totalImpressions = 1;
       } else {
-        parts = total.split(","),
+        parts = total.split("_"),
```

Comment:
total preference thing but you can also use `|` i believe. not a big deal since we control shit cookie but u are less likely to run into a pipe in key names and stuff like that in the grand scheme of cookie values. 

it also might make sense to just make a function to encode our cookies and decode them coming back out so they can have whatever values we want...again not a big deal for this fix but something to think about.